client/scripts/controllers/server/threads.ts

```diff
@@ -148,6 +148,7 @@ class ThreadsController {
     proposal: OffchainThread,
     body?: string,
     title?: string,
+    mentions?: string[],
```

Comment:
It seems like we should parse out mentions on the backend, not the frontend, rather than trusting the frontend to handle this correctly.

Reply:
Okay, we can do that! I can try moving all the algorithms for parsing to the server utils

Reply:
Let me put a separate PR in for that though, since it goes beyond the scope of just the editThread route